test/runner/lib/executor.py

```diff
@@ -19,6 +18,11 @@
 import string
 import shutil
 
+try:
+    from shlex import quote as cmd_quote
+except ImportError:
+    from pipes import quote as cmd_quote
+
```

Comment:
Instead of repeating the import try/except here, just add `cmd_quote` to the `from lib.util import` below.

Reply:
Ah, missed that. Thanks.